app/controllers/v0/claim_letters_controller.rb

```diff
@@ -26,7 +26,12 @@ def show
     private
 
     def service
-      if Flipper.enabled?(:cst_claim_letters_use_lighthouse_api_provider, @current_user)
+      use_lighthouse = Flipper.enabled?(:cst_claim_letters_use_lighthouse_api_provider, @current_user)
+      api_provider = use_lighthouse ? 'lighthouse' : 'VBMS'
+      ::Rails.logger.info('Choosing Claim Letters API Provider via cst_claim_letters_use_lighthouse_api_provider',
+                          { message_type: 'cst.api_provider',
+                            api_provider: })
```

Comment:
Are we missing the api_provider value here?  

`Rails.logger.info('Choosing Claim Letters API Provider via cst_claim_letters_use_lighthouse_api_provider',
                  { message_type: 'cst.api_provider',
                    api_provider: api_provider })`

Reply:
no, ruby (3.1+) will destructure and place in the value of the above variable in that payload hash.

Reply:
Good to know. I was wondering if there was some sort of magic going on there.